state_formatters/dp_formatters.py

```diff
@@ -1252,3 +1255,56 @@ def dff_command_selector_skill_formatter(dialog: Dict) -> List[Dict]:
     batches[-1]["dialog_batch"][-1]["called_intents"] = called_intents
     batches[-1]["dialog_batch"][-1]["dialog_id"] = dialog.get("dialog_id", "unknown")
     return batches
+
+
+def user_emotion_bot_mood_formatter(dialog: Dict) -> List[Dict]:
+    # comment out the next two lines to save dialog data to see its contents
+    # with open('test_formatters.json', 'w', encoding='utf-8') as f:
+    #     json.dump(dialog, f, ensure_ascii=False, indent=4)
+    sentences = [dialog["human_utterances"][-1]["text"]]
+    annotated_utterances = [dialog["human_utterances"][-1]]
+
+    if len(dialog["bot_utterances"]) > 1:
+        bot_mood = (
+            dialog["bot_utterances"][-2]["annotations"]
+            .get("bot_emotion_classifier", {})
+            .get("bot_mood", [0.75, 0.25, 0.44])
```

Comment:
why the numbers are here? what is it?

Reply:
this is default mood of the bot.
added comments about it